examples/demo.js

```diff
@@ -207,6 +209,44 @@ window.onload = function () {
     });
   }
 
+  /**
+  * Get the data URL for the last image in the sequence.
+  * @return {string} The data URL for the last image in the sequence.
+  */
+  function getLastImage() {
+    // Get the image from the last step.
+    let imgs = document.getElementsByClassName('step-thumbnail');
+    let lastStepImage = imgs[imgs.length-1];
+    return lastStepImage.getAttribute("src");
+  }
+
+  /**
+  * Download the given image URL as a PDF file.
```

Comment:
Remove full stop from phrases.

Reply:
Isn't it a complete sentence?

Reply:
It seems like the [JSDoc examples](https://devdocs.io/jsdoc/about-block-inline-tags) usually look like this (they use imperative sentences).